### PR TITLE
fixes #266 (Axe failureSummary leaks English strings)

### LINKED ISSUE #266: Question - possible non localized strings
These two strings are not being localized even when using a language available for both Ace and Axe:

`Fix any of the following / Fix all of the following`

They appear on the report and I'm not sure either they are issued by Ace or Axe.
Looking at the code below, I can see they have been removed from Axe messages, but I couldn't find a insertion point later, which I presume to exist since the strings are present both on Html and Json reports.

https://github.com/daisy/ace/blob/bf8b37442d5557d056d131057a8213ba24334c6e/packages/ace-report-axe/src/index.js#L106-L117

### PR DESCRIPTION
**Pull request by danielweck**

### CODE REVIEW

File: packages/ace-report-axe/src/index.js [hunk 1]
```diff
@@ -105,18 +105,21 @@ function  axe2ace(spineItem, axeResults, lang) {
...
-        description = description.replace(/[^:]+:/, "");
+        // Corriger l’un des éléments suivants :
 
+        // TODO: this is a hacky way to fix the Axe bug that forces the wrong language (en) into `failureSummary`.
+        description = description.replace(/Fix any of the following:/g, "");
+        description = description.replace(/Fix all of the following:/g, "");
         description = description.trim();
+        description = description.replace(/[\n]+/g, "\n");
+        description = description.replace(/\n/g, " --- ");
+        description = description.replace(/\s+/g, " ");
```
Comment (thiagoeec) [inline packages/ace-report-axe/src/index.js]: @danielweck 

I've tested it here, and it works. I just have a suggestion: if you remove the last three lines of regex, then you retain the former design of the report (bullets), which looked better, in my opinion.

Comment (danielweck) [reply packages/ace-report-axe/src/index.js]: Yes, you are correct, the HTML report looks bad. I fixed the normalization function for the `dct:description` JSON field, but I must also ensure the contents are rendered well in the HTML markup. I will be looking into this shortly.

Comment (danielweck) [reply packages/ace-report-axe/src/index.js]: Actually this was a "typo" on my part, thank you for catching this! Correcting now.

Comment (danielweck) [reply packages/ace-report-axe/src/index.js]: Done: https://github.com/daisy/ace/commit/6f624bcef18e59fffe46ec63f658b61e37e6435d
Sorry about that, the collapsing of whitespaces included line breaks, and I forgot to re-add them as separators. Thank you for checking, much appreciated!

**Comment by danielweck** [comment]
Follow-up hotfix: https://github.com/daisy/ace/commit/6f624bcef18e59fffe46ec63f658b61e37e6435d

**Comment by thiagoeec** [comment]
Nice! Thanks, @danielweck .

**Pull request merged**